### PR TITLE
【InfRT】fix test error

### PR DESCRIPTION
**Pull request by shangzhizhou**
### PR types
Others

### PR changes
Others

### Describe
trt-exec测试示例修复

### CODE REVIEW

File: paddle/infrt/dialect/tensorrt/trt_exec.cc [hunk 1]
```diff
@@ -32,17 +32,34 @@ int main(int argc, char** argv) {
...
+    std::cout << "\ntrtGraphFusePass pass failed!\n" << std::endl;
+    return 4;
+  }
+  std::cout << "\nAfter trtGraphFusePass IR dump:\n";
+  module->dump();
+
   trt_pass_manager.addPass(std::make_unique<infrt::trt::trtGraphSplitPass>(10));
   if (mlir::failed(pm.run(*module))) {
-    std::cout << "\npass failed!\n" << std::endl;
+    std::cout << "\ntrtGraphSplitPass pass failed!\n" << std::endl;
```
Comment (Superjomn) [inline paddle/infrt/dialect/tensorrt/trt_exec.cc]: No need to specify the name of the pass.

Many other differnet pass will register to system latter, not just this pass.

**Review by winter-wang — APPROVED** [review]
LGTM

File: paddle/infrt/dialect/tensorrt/trt_exec.cc [hunk 2]
```diff
@@ -32,17 +32,34 @@ int main(int argc, char** argv) {
   mlir::MLIRContext* context = infrt::Global::getMLIRContext();
   auto module = infrt::dialect::LoadMlirFile(input_file.c_str(), context);
 
+  std::cout << "Input IR dump:\n";
```
Comment (Superjomn) [inline paddle/infrt/dialect/tensorrt/trt_exec.cc]: Remove this, no need to display the input IR.

The llvm lit will exemine the stdout, so avoid print unnecessary infomation to the stdout.

File: paddle/infrt/dialect/tensorrt/trt_exec.cc [hunk 3]
```diff
@@ -32,17 +32,34 @@ int main(int argc, char** argv) {
...
   module->dump();
   mlir::PassManager pm(context);
 
   mlir::OpPassManager& trt_pass_manager = pm.nest<mlir::FuncOp>();
   trt_pass_manager.addPass(std::make_unique<infrt::trt::trtOpTellerPass>());
+  if (mlir::failed(pm.run(*module))) {
+    std::cout << "\ntrtGraphFusePass pass failed!\n" << std::endl;
+    return 4;
+  }
+  std::cout << "\nAfter trtOpTellerPass IR dump:\n";
```
Comment (Superjomn) [inline paddle/infrt/dialect/tensorrt/trt_exec.cc]: Remove this line.

The exec command line can config which pass to execute, so this line is unnecessary when running other passes.

File: paddle/infrt/dialect/tensorrt/trt_exec.cc [hunk 4]
```diff
@@ -32,17 +32,34 @@ int main(int argc, char** argv) {
...
+  module->dump();
+
   trt_pass_manager.addPass(std::make_unique<infrt::trt::trtGraphFusePass>());
+  if (mlir::failed(pm.run(*module))) {
```
Comment (Superjomn) [inline paddle/infrt/dialect/tensorrt/trt_exec.cc]: No need to dump twice.

**Review by Superjomn — COMMENTED** [review]
Only the disabled_trt_ops.mlir modification is needed.

File: paddle/infrt/tests/dialect/disabled_trt_ops.mlir [hunk 5]
```diff
@@ -7,15 +7,15 @@ func @main() -> tensor<?xf32> {
   %bias1 = "pd.feed"() {name="input4"} : () -> tensor<?xf32>
   %bias2 = "pd.feed"() {name="input5"} : () -> tensor<?xf32>
 
-  %d = "pd.elementwise_add"(%c, %bias) {axis=1:i32} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
+  %d = "pd.elementwise_add"(%c, %bias) {axis=1:si32} : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
```
Comment (jiweibo) [inline paddle/infrt/tests/dialect/disabled_trt_ops.mlir:10]: 是否能改下OP定义，支持i32的传入呢

File: paddle/infrt/dialect/tensorrt/trt_exec.cc (hunk 4 above)
Comment (shangzhizhou) [reply paddle/infrt/dialect/tensorrt/trt_exec.cc]: 这个文件的改动已经revert

File: paddle/infrt/tests/dialect/disabled_trt_ops.mlir (hunk 5 above)
Comment (shangzhizhou) [reply paddle/infrt/tests/dialect/disabled_trt_ops.mlir:10]: attribute只能是确定的类型，可以后面看看是否可以load的时候自动转换兼容类型

**Pull request closed without merging**